### DISCUSSION TITLE
Please add responsive height

### DISCUSSION THREAD

**Discussion by ShourovRoy** [tailwindlabs/tailwindcss#4478] [Ideas]
Tailwind is really helpful for making responsive websites but there is a lack of feature which is responsive height. Please add it.

**Comment by longtimedeveloper** [comment]
@ShourovRoy responsive height is working for me.  The `h-96` is the responsive class.  Hope this helps, Karl

`<section class="flex items-center justify-center h-96 m-auto mb-12 bg-fixed bg-center bg-cover bg-location">`

**Reply by longtimedeveloper** [reply]
@ShourovRoy 

Here is a better example.  You can clearly see the height is responsive and changes at the large screen size.

`<section class="flex flex-col h-80 lg:h-96">`

Karl

**Comment by simonswiss** [comment]
Hey!

What do you mean by responsive height? A height that is expressed in % as we do for width, like `h-1/2`, `h-1/3`, etc?

Or a height that is proportional to the viewport, like `h-screen-1/2`?
